pkg/install/role.yaml

```diff
@@ -78,6 +78,9 @@ rules:
   - verbs: ["get", "list", "watch"]
     apiGroups: ["storage.k8s.io"]
     resources: ["storageclasses"]
+  - verbs: ["*"]
```

Comment:
Can this just be `["get", "list", "watch"]`?

Reply:
Yes, it can.